docs/AboutUs.md

```diff
@@ -41,8 +41,8 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 
 <img src="images/shawn532.png" width="200px">
 
-[[github](http://github.com/shawn532)]
-[[portfolio](team/johndoe.md)]
+[github](http://github.com/shawn532)
+[portfolio](team/yuxuanleong.md)
```

Comment:
Sorry I forgot to change this. But should this be shawn532.md instead? I will change it afterwards. Besides, Thank you for adding a userguide-screenshots folder and moving my shawn532.png to the right folder.